docs/3.0.0-beta.x/admin-panel/customization.md

```diff
@@ -10,19 +10,19 @@ To apply your changes you need to [rebuild](#build) your admin panel
 
 By default, the administration panel is exposed via [http://localhost:1337/admin](http://localhost:1337/admin). However, for security reasons, you can easily update this path.
 
-**Path —** `./config/environment/**/server.json`.
-
-```json
-{
-  "host": "localhost",
-  "port": 1337,
-  "cron": {
-    "enabled": false
+**Path —** `./config/server.js`.
```

Comment:
Is the per environment server.json file being depreciated? If so, it needs to go into migration guide.

Reply:
It's not deprecated per say but change. By now there is the default server.js config so I think we should only document those and explain in the config how a user can customize per env (done in the PR). 

And Yes there will be a big migration guide / script for this ;)

Reply:
Gotcha :+1: